src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,18 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+
+public class FindCommand extends Command {
+    private String matchString;
+
+    public FindCommand(String matchString) {
+        this.matchString = matchString;
+    }
+    @Override
+    public void execute(TaskList list, Storage storage) throws DukeException {
```

Comment:
Will not be commenting on this any further. Hope that my previous comments on execute will help your code clarity 😃

Reply:
Thanks for pointing it out! I did want to make the method name more descriptive, but it extends from the Command abstract class, which as the execute method, so I need to implement a method that has the same method signature, so it needs to have the same name "execute".